src/olympia/pages/templates/pages/shield_study_2.html

```diff
@@ -26,7 +29,7 @@
       <div class="details-header content-wrapper">
         <h1 data-hook="title">Help Make Firefox Faster</h1>
         <div class="experiment-controls">
-          <a class="button primary large" href="https://addons.mozilla.org/firefox/downloads/latest/x-screen-performance-study-1/">Start Performance Study</a>
+          <span id="opt-in" class="button primary large">Start Performance Study</span>
```

Comment:
Is there a reason to not have a real link here? It might be nice to link through to something even if JS was disabled. Also spans aren't focusable for people that use a keyboard to navigate.